source/js/parse-iiif-manifest.js

```diff
@@ -98,9 +106,9 @@ export default function parseIIIFManifest (manifest)
     for (let i = 0; i < numCanvases; i++)
     {
         thisCanvas = canvases[i];
-        canvas = thisCanvas['@id'];
+        canvas = thisCanvas['@id'] || thisCanvas.id;
         label = thisCanvas.label;
-        thisResource = thisCanvas.images[0].resource;
+        thisResource = thisCanvas.images ? thisCanvas.images[0].resource : thisCanvas.items[0].items[0].body;
```

Comment:
We should make sure we check this against the multiple images per canvas feature. See L141 below.